web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -140,26 +140,24 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
       },
       approvalRequired: true,
       image: userUtils.defaultAvatar('champion'),
-      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      text: $translate.instant('Youth Champion Text...'),
```

Comment:
a long string is no more a string... :p 
cd-desc sounds good to me; but I'm asking myslef if we're not overloading the role of translation file with a context, in a way we're going to have keys in this service that serve strings, while we could have keys in the translations and a context to describe the english "wording" ? Problem could be to invalidate a key when you change the context (aka, the content of the string)